### PR TITLE
Search application results page

### PR DESCRIPTION
**Pull request by samfallowfield**
Created the search application results page.

Currently without Pagination or links to individual reports via reference number

### CODE REVIEW

File: CheckYourEligibility-Admin/Views/Application/Results.cshtml [hunk 1]
```diff
@@ -1,12 +1,70 @@
-﻿@model CheckYourEligibility_FrontEnd.Models.ApplicationSearch;
+﻿@model CheckYourEligibility.Domain.Responses.ApplicationSearchResponse;
+
+@{
+    Dictionary<string, string> statusColor = new Dictionary<string, string>
```
Comment (RichC24789) [inline CheckYourEligibility-Admin/Views/Application/Results.cshtml:4]: why is this called color

File: CheckYourEligibility-Admin/Views/Application/Results.cshtml [hunk 2]
```diff
@@ -1,12 +1,70 @@
...
+                       <td class="govuk-table__cell">@application.Reference</td>
+                       <td class="govuk-table__cell">@(@application.ParentFirstName + " " + @application.ParentLastName)</td>
+                       <td class="govuk-table__cell">@(@application.ChildFirstName + " " + application.ChildLastName)</td>
+                       <td class="govuk-table__cell">@Convert.ToDateTime(application.ChildDateOfBirth).ToString("MMM dd yyyy")</td>
+                       <td class="govuk-table__cell">
+                            <strong class="govuk-tag @statusColor[application.Status]">
+                                @statusName[application.Status]
+                            </strong>
+                       </td>
+                       <td class="govuk-table__cell">@application.Created.ToString("MMM dd yyyy")</td>
```
Comment (RichC24789) [inline CheckYourEligibility-Admin/Views/Application/Results.cshtml:62]: is this date format correct?

**Review by RichC24789 — CHANGES_REQUESTED** [review]
There should be covering tests

File: CheckYourEligibility-Admin/Views/Application/Results.cshtml (hunk 2 above)
Comment (samfallowfield) [reply CheckYourEligibility-Admin/Views/Application/Results.cshtml:62]: yes based on the current prototype

Comment (RichC24789) [reply CheckYourEligibility-Admin/Views/Application/Results.cshtml:62]: Double check with Adam/deepa

**Review by RichC24789 — DISMISSED** [review]

**Review by RichC24789 — APPROVED** [review]

**Pull request merged**